test/rulesets/Base/base.jl

```diff
@@ -99,6 +99,18 @@
         end
     end
 
+    @testset "Unary complex functions" begin
+        for x in rand.((Int, Float32, Float64, complex.((Float32, Float64))...))
```

Comment:
```suggestion
        for x in rand.((Int32, Int64, Float32, Float64, Complex{Float32}, Complex{Float64}))
```

Reply:
also, is it a problem that rand(Int) is going to give an "extreme" number
```julia
julia> rand(Int32)
507053106

julia> rand(Int64)
7317803238138782059
```

Reply:
Probably best just to remove the integer tests, or at least hard code them to more reasonable values